src/wallet/rpcwallet.cpp

```diff
@@ -400,12 +400,21 @@ UniValue SendMoney(CWallet* const pwallet, const CCoinControl &coin_control, std
     CAmount nFeeRequired = 0;
     int nChangePosRet = -1;
     bilingual_str error;
-    CTransactionRef tx;
-    bool fCreated = pwallet->CreateTransaction(recipients, tx, nFeeRequired, nChangePosRet, error, coin_control, !pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS));
+    CTransactionRef tx; 
+    std::string fee_reason;
+    bool fCreated = pwallet->CreateTransaction(recipients, tx, nFeeRequired, nChangePosRet, error, coin_control, fee_reason, !pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS));
     if (!fCreated) {
         throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, error.original);
     }
     pwallet->CommitTransaction(tx, std::move(map_value), {} /* orderForm */);
+
+    if(verbose){
+        UniValue entry(UniValue::VOBJ);
+        entry.pushKV("hex", tx->GetHash().GetHex());
```

Comment:
`hex` usually implies the full raw transaction. This is just a txid, so this should instead be named `txid`.